src/app/docs/split-view/split-view-docs.component.html

```diff
@@ -8,9 +8,146 @@
+              <sky-definition-list-value>
+                {{ activeRecord.date }}
+              </sky-definition-list-value>
+            </sky-definition-list-content>
+            <sky-definition-list-content>
+              <sky-definition-list-label>
+                Vendor
+              </sky-definition-list-label>
+              <sky-definition-list-value>
+                {{ activeRecord.vendor }}
+              </sky-definition-list-value>
+            </sky-definition-list-content>
+            <sky-definition-list-content>
+              <sky-definition-list-label>
+                Receipt image
+              </sky-definition-list-label>
+              <sky-definition-list-value>
+                {{ activeRecord.receiptImage }}
+              </sky-definition-list-value>
+            </sky-definition-list-content>
+          </sky-definition-list>
+          <div
+            class="sky-form-group"
+          >
+            <label
+              class="sky-control-label"
+              for="approvedAmount"
+            >
+              Approved amount
+            </label>
+            <input
+              class="sky-form-control"
+              formControlName="approvedAmount"
+              name="approvedAmount"
+              type="text"
+            />
+          </div>
+
+          <div
+            class="sky-form-group"
+          >
+            <label
+              class="sky-control-label"
+              for="comments"
+            >
+              Comments
+            </label>
+            <textarea
+              class="sky-form-control"
+              formControlName="comments"
+              name="comments"
+            ></textarea>
+          </div>
+        </form>
+      </sky-split-view-workspace-content>
+      <sky-split-view-workspace-footer>
+        <sky-summary-action-bar
+          id="summary-action-bar"
+        >
+          <sky-summary-action-bar-actions>
+            <sky-summary-action-bar-primary-action
+              (actionClick)="onApprove()"
+            >
+              Approve expense
+            </sky-summary-action-bar-primary-action>
+            <sky-summary-action-bar-secondary-actions>
+              <sky-summary-action-bar-secondary-action
+                (actionClick)="onDeny()"
+              >
+                Deny request
+              </sky-summary-action-bar-secondary-action>
+            </sky-summary-action-bar-secondary-actions>
+          </sky-summary-action-bar-actions>
+        </sky-summary-action-bar>
+      </sky-split-view-workspace-footer>
+    </sky-split-view-workspace>
+  </sky-split-view>
   </sky-docs-demo>
 
+  <sky-docs-code-examples
+    [packageDependencies]="{
+      '@skyux/action-bars': '*',
+      '@skyux/inline-form': '*',
+      '@skyux/lists': '*',
+      '@skyux/split-view': '*'
+    }"
+  >
+    <sky-docs-code-example
+      heading="Basic setup"
+      sourceCodePath="src/app/public/plugin-resources/code-examples/split-view/basic"
```

Comment:
Does this need a trailing slash?
```suggestion
      sourceCodePath="src/app/public/plugin-resources/code-examples/split-view/basic/"
```

Reply:
Doesn't appear so? Everything renders fine. I can add it if you want but everything is showing up ok.